components/GM.js

```diff
@@ -0,0 +1,177 @@
+// import React from 'react';
```

Comment:
Why is this file completely commented out?

Reply:
I think GM needs to be refactored into an actual component in a dummy page, will probably be another issue!
@mattxwang should have more info

Reply:
Yeah, just to follow up - since the page isn't live, didn't want to introduce more complexity into the change. Eventually, we should adjust this too.

A quick thought - I think the page for a GM (ex `gm-w21`) should wrap the existing `GM` component, and simply pass in props to the component. That way, the `GM` component itself doesn't need to rely on external data sources, which may be a better separation of concerns.